CHANGELOG.md

```diff
@@ -10,6 +10,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+## Breaking Changes
+- liboecryptombed is renamed to liboecryptombedtls.
+- The `openenclave-config.cmake` and `openenclave-lvi-mitigation-config.cmake` no longer include liboecryptombedtls as `PUBLIC` link requirement for liboeenclave.
```

Comment:
It might help to link the two statements:
```suggestion
- liboecryptombed is now called liboecryptombedtls and will no longer be automatically included as a link dependency when linking liboeenclave in CMake.
   - The `openenclave-config.cmake` and `openenclave-lvi-mitigation-config.cmake` will not specify the renamed liboecryptombedtls as a `PUBLIC` link requirement for liboeenclave.
```

Reply:
Updated.